src/System.Collections.Concurrent/tests/ProducerConsumerCollectionTests.cs

```diff
@@ -735,73 +735,35 @@ public void ManyConcurrentAddsTakes_EnsureTrackedCountsMatchResultingCollection(
             }
         }
 
-        [Theory]
-        [InlineData(3000000)]
+        [Fact]
         [OuterLoop]
-        public void ManyConcurrentAddsTakes_CollectionRemainsConsistent(int operations)
+        public void ManyConcurrentAddsTakes_CollectionRemainsConsistent()
         {
             IProducerConsumerCollection<int> c = CreateProducerConsumerCollection();
 
-            // Thread that adds
-            Task<HashSet<int>> adds = ThreadFactory.StartNew(() =>
+            const int operations = 30000;
+            Action addAndRemove = () =>
             {
-                for (int i = int.MinValue; i < (int.MinValue + operations); i++)
+                for (int i = 1; i < operations; i++)
```

Comment:
No reason. The loop I modified started the index at `1` because it was adding the index value itself to the collection and wanted to avoid using `default(int)`.